other/materials_designer/create_nanowire_custom_shape.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>We should isolate the rotation to its own cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>